tests/test_torch_agent.py

```diff
@@ -320,7 +320,6 @@ def test_batchify(self):
             self.assertIsNone(batch.label_vec)
             self.assertIsNone(batch.label_lengths)
             self.assertIsNone(batch.labels)
-            self.assertIsNone(batch.valid_indices)
```

Comment:
why remove these?

Reply:
Previously we checked the batch for text_vec to determine if the observations were padding or not. Since we were batchifying without observing, there was no text_vec and so all were padding, therefore a None valid_indices.

Now we check for the sentinel key `is_padding` or whatever. This no longer depends on whether we've tokenized or not. Therefore, `valid_indices` ends up getting set accurately, though the other fields are still empty